src/components/backend-ai-console.ts

```diff
@@ -1111,7 +1111,7 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
         ${this._offline ? _t("console.YouAreOffline") : _t("console.YouAreOnline")}.
       </backend-ai-offline-indicator>
       <backend-ai-login active id="login-panel"></backend-ai-login>
-      <backend-ai-splash id="about-panel"></backend-ai-splash>
+      <backend-ai-splash id="about-backend.ai-panel"></backend-ai-splash>
```

Comment:
@swapniljha001 (Same as above) We need to avoid using dot (.) at the selector name (even it works). please change it to backendai. :)

Reply:
Fixed.